hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieAppendHandle.java

```diff
@@ -164,6 +164,7 @@ private void init(HoodieRecord record) {
         // Since the actual log file written to can be different based on when rollover happens, we use the
         // base file to denote some log appends happened on a slice. writeToken will still fence concurrent
         // writers.
+        // TODO? are these sufficient?
```

Comment:
the cleanest way to have a marker per file created. That way we can avoid listing anything at all. We can just compute the files from the marker folder directly

Reply:
lets have a follow on JIRA for this. We should ideally fix so that marker files exist for all log files. and we are able to just add the actual writtenLogFiles ..

Reply:
created a follow up ticket https://issues.apache.org/jira/browse/HUDI-1517. not fixing in this release